grpc_http.go

```diff
@@ -0,0 +1,144 @@
+package grpc
```

Comment:
Underscore has a special meaning in Go filenames (build tags). The filename should be `grpchttp`.